lib/modules/manager/npm/update/package-version/index.ts

```diff
@@ -19,10 +19,10 @@ export function bumpPackageVersion(
       const mirrorPackage = bumpVersion.replace('mirror:', '');
       const parsedContent = JSON.parse(content);
       newPjVersion =
-        (parsedContent.dependencies || {})[mirrorPackage] ||
-        (parsedContent.devDependencies || {})[mirrorPackage] ||
-        (parsedContent.optionalDependencies || {})[mirrorPackage] ||
-        (parsedContent.peerDependencies || {})[mirrorPackage];
+        parsedContent.dependencies?.mirrorPackage ||
+        parsedContent.devDependencies?.mirrorPackage ||
+        parsedContent.optionalDependencies?.mirrorPackage ||
+        parsedContent.peerDependencies?.mirrorPackage;
```

Comment:
@rarkins this somehow changed snapshot, we maybe need to use `??` instead of `||`?

Reply:
```suggestion
        parsedContent.dependencies?.[mirrorPackage] ??
        parsedContent.devDependencies?.[mirrorPackage] ??
        parsedContent.optionalDependencies?.[mirrorPackage] ??
        parsedContent.peerDependencies?.[mirrorPackage];
```